### PR TITLE
[Dev Portal] Help side panel feature

### LINKED ISSUE #593: [Dev Portal] Help should be provided to the users while performing dev portal tasks
**Is your feature request related to a problem? Please describe.**
Inexperienced users might feel confused while performing tasks from the developer portal without proper guidance.

**Describe the solution you'd like**
Users should be given the necessary insight to perform tasks easily and smoothly from the developer portal.

Ex. If a user is editing a service provider, it would be nice if the docs and other useful supplementary resources such as blogs, links, etc. related to service provider configurations are provided to the user from inside the portal to improve UX.

### PR DESCRIPTION
**Pull request by brionmario**
## Purpose
Inexperienced users might feel confused while performing tasks from the developer portal without proper guidance.

## Goals
This feature will introduce a help side panel which will provide the user with useful resources such as docs, artifacts etc.

Resolves #593 

## Approach
As part of development, reusable components for side panel and mark down renderer, repo card was created in `wso2is/react-components` module.

Screen Shot 2020-04-09 at 5 47 08 AM
Screen Shot 2020-04-09 at 5 47 31 AM
Screen Shot 2020-04-09 at 5 48 43 AM
Screen Shot 2020-04-09 at 5 48 04 AM

### CODE REVIEW

File: modules/i18n/src/translations/ta-IN/portals/common.ts [hunk 1]
```diff
@@ -43,7 +43,10 @@ export const common: CommonNS = {
     delete: "அழி",
     description: "விபரம்",
     deviceModel: "கருவி மாதிரி",
+    docs: "டாக்ஸ்",
```
Comment (thivi) [inline modules/i18n/src/translations/ta-IN/portals/common.ts:46]:
```suggestion
    docs: "உதவி ஆவணங்கள்",
```

File: modules/i18n/src/translations/ta-IN/portals/common.ts [hunk 2]
```diff
@@ -78,7 +81,9 @@ export const common: CommonNS = {
     retry: "மீண்டும் முயற்சி செய்",
     revoke: "நீக்கு",
     revokeAll: "அனைத்தையும் நீக்கு",
+    samples: "மாதிரிகள்",
     save: "சேமி",
+    sdks: "SDKs",
```
Comment (thivi) [inline modules/i18n/src/translations/ta-IN/portals/common.ts:86]:
```suggestion
    sdks: "SDKகள்",
```

File: apps/admin-portal/src/pages/application-edit.tsx [hunk 3]
```diff
@@ -43,12 +62,66 @@ export const ApplicationEditPage: FunctionComponent<{}> = (): ReactElement => {
...
+     * Called when help panel doc URL status changes.
+     */
+    useEffect(() => {
+        if (!helpPanelDocURL) {
+            return;
+        }
+
+        setHelpPanelDocContentRequestLoadingStatus(true);
+
+        fetch(helpPanelDocURL)
```
[the remark this answers is not in the document]
Comment (brionmario) [reply apps/admin-portal/src/pages/application-edit.tsx]: Fixed with `57e258d`

File: modules/i18n/src/translations/ta-IN/portals/common.ts (hunk 1 above)
Comment (brionmario) [reply modules/i18n/src/translations/ta-IN/portals/common.ts:46]: Fixed with `57e258d`

File: modules/i18n/src/translations/ta-IN/portals/common.ts (hunk 2 above)
Comment (brionmario) [reply modules/i18n/src/translations/ta-IN/portals/common.ts:86]: Fixed with `57e258d`

**Pull request merged**